flight/Modules/ManualControl/transmitter_control.c

```diff
@@ -249,6 +249,7 @@ int32_t transmitter_control_update()
 
 		if (settings.ChannelGroups[n] >= MANUALCONTROLSETTINGS_CHANNELGROUPS_NONE) {
 			cmd.Channel[n] = PIOS_RCVR_INVALID;
+			validChannel[n] = false;
```

Comment:
>  on an error code and then use that scaled version as input to validInputRange() 

I don't think the scaled version is input to validInputRange.

It also won't be negative because everything is coerced to a uint16.